### PR TITLE
[1.11] introduce semantic-release

### PR DESCRIPTION
**Pull request by juliangieseke**
This backports our `semantic-release` setup back to 1.11 branch. `standard-version` 4.3.0 has a bug related to our tag format and also it includes _way_ too much changes in the changelog, Version 4.4.0 hangs on my machine so I checked if it is possible to easily backport `semantic-release` to 1.11 branch - looks like it is 👍 

## Testing

merge -> check master ci 🤷‍♂️ 

## Trade-offs

its a plain backport, nothing changed

### CODE REVIEW

File: scripts/ci/utils/upload-sourcemaps [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```
Comment (juliangieseke) [inline scripts/ci/utils/upload-sourcemaps]: @nLight anything else we have to do to enable it on 1.11 branch? Or shall we extract it from this PR for now?

**Comment by GeorgiSTodorov** [comment]
File conflicts.

File: scripts/ci/utils/upload-sourcemaps (hunk 1 above)
Comment (nLight) [reply scripts/ci/utils/upload-sourcemaps]: It should be good. Basically Sentry only cares about the release and source maps. If they are there it's good to go

**Comment by juliangieseke** [comment]
rebased /cc @nLight @GeorgiSTodorov

**Comment by juliangieseke** [comment]
trying to get CI green… ⏳

**Pull request merged**